platforms/documentation/docs/src/docs/userguide/api/kotlin_dsl.adoc

```diff
@@ -908,7 +908,11 @@ Some plugins expect simple closures, as with the link:{plugin-portal}plugin/com.
 
 .Use `closureOf<T> {}`
 ====
-include::sample[dir="snippets/kotlinDsl/interoperability-closure-of/kotlin",files="build.gradle.kts[tags=closureOf]"]
+bintray {
```

Comment:
Let's do this to unblock us, but we really should find another example.

Reply:
FTR: AFAICT tracked in https://github.com/gradle/gradle/issues/29955